### PR TITLE
Fixes #9868 - adding tests for organization controller

### PR DESCRIPTION
**Pull request by lizagilman**

### CODE REVIEW

File: test/controllers/api/v2/organizations_controller_test.rb [hunk 1]
```diff
@@ -81,4 +81,14 @@ class Api::V2::OrganizationsControllerTest < ActionController::TestCase
     assert_response :success
     assert_equal 1, organization.reload.organization_parameters.count
   end
+
+  test "should not update invalid organization" do
+    put :update, params: { :id => Organization.first.to_param, :organization => { :name => "" } }
```
Comment (lzap) [inline test/controllers/api/v2/organizations_controller_test.rb]: Hey, the `to_param` method returns `/organiazation/1` while here you want to put just number instead. Try `Organization.first.id` instead.

**Review by lzap — CHANGES_REQUESTED** [review]
Thanks one comment.

**Comment by lizagilman** [comment]
@lzap thank you, updated

**Comment by lizagilman** [comment]
[test foreman]
[test katello]

**Review by lzap — APPROVED** [review]
Thank you!

**Pull request merged**